docs/cloud/reviews/multi-snapshot.mdx

```diff
@@ -1,6 +1,6 @@
 # Multi Snapshot Reviews
```

Comment:
What do you think about including this [video sequence](https://youtu.be/GJGGhUTT7LE?feature=shared&t=2941) from 49:01 - 51:17?

Reply:
Nice idea 🎉